src/pages/[platform]/build-a-backend/graphqlapi/best-practice/index.mdx

```diff
@@ -0,0 +1,41 @@
+
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+import { getChildPageNodes } from '@/utils/getChildPageNodes';
+import directory from 'src/directory/directory.json';
+
+export const meta = {
+    title: 'Best Practice',
+    description: 'Best Practice',
+    platforms: [
```

Comment:
Just used a script to generate these files, the title and route are simple enough to get based on the file path this platforms list however is always just using all platforms.  How can we determine this platform list for one of these "Summary" pages?